AAD Token support (#6282)
Azure/azure-sdk-for-java: sdk/appconfiguration/azure-data-appconfiguration/README.md

        .buildAsyncClient();
```

You can also use `TokenCredential` to create a configuration client, such as an Azure Active Directory (AAD) token.
Unlike a connection string if you're using an AAD token you must supply the endpoint of AppConfiguration service. The
endpoint can be obtained by going to your App Configuration instance in the Azure portal and navigating to "Overview"

> page. Looking for the "Endpoint" keyword. 

Reviewer: ```suggestion
page and look for the "Endpoint" keyword. 
```

        .buildAsyncClient();
```

> You can also use `TokenCredential` to create a configuration client, such as AAD token. 

Reviewer: ```suggestion
You can also use `TokenCredential` to create a configuration client, such as an AAD token. 
```

> If you using to AAD, make sure you have the endpoint of AppConfiguration service. You can find the end point from 

Reviewer: ```suggestion
Unlike a connection string if you're using an AAD token you must supply the endpoint of AppConfiguration service. You can find the end point from 
```

> connection string itself by looking for `endpoint={endpoint_value};`. You can get `TokenCredential` from other azure service

Reviewer: I think we should give them the example of using the portal to find the AppConfiguration endpoint, if we say be that you can use a connection string to authenticate and say here use a connection string to find the endpoint I won't use AAD since I could just use a connection string and remove the hassle of using AAD.

        .buildAsyncClient();
```

> You can also use `TokenCredential` to create a configuration client, such as an AAD token. 

Author: Azure Activity Directory

> You can also use `TokenCredential` to create a configuration client, such as an AAD token. 

Reviewer 2: Yes. I meant to expand on this "such as an Azure Active Directory (AAD) token."

Unlike a connection string if you're using an AAD token you must supply the endpoint of AppConfiguration service. The
endpoint can be obtained by going to your App Configuration instance in the Azure portal and navigating to "Overview"

> page. Looking for the "Endpoint" keyword.. You can get `TokenCredential` from other azure service

Reviewer: ```suggestion
page. Looking for the "Endpoint" keyword. You can get `TokenCredential` from other Azure services
```

[app_config_store]: https://docs.microsoft.com/azure/azure-app-configuration/quickstart-dotnet-core-app#create-an-app-configuration-store
[azconfig_docs]: https://docs.microsoft.com/azure/azure-app-configuration
[azure_cli]: https://docs.microsoft.com/cli/azure

> [azure_identity]: https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-overview

Reviewer 2: Remove `en-us`. User's browser can determine the language.
